pkg/client/add_manifest.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+import (
+	"context"
+	"fmt"
+)
+
+type ManifestAddOptions struct {
+	// Image index we want to update
+	IndexRepoName string
+
+	// Name of image we wish to add into the image index
+	RepoName string
+}
+
+// AddManifest implements commands.PackClient.
+func (c *Client) AddManifest(ctx context.Context, opts ManifestAddOptions) (err error) {
```

Comment:
Similar comment to https://github.com/buildpacks/pack/pull/1705#discussion_r1586320093